pages/docs/reference/mpp-set-up-targets.md

```diff
@@ -0,0 +1,78 @@
+---
+type: doc
+layout: reference
+title: "Set up targets manually"
+---
+
+# Set up targets manually
+
+You can add targets when [creating a project with the Project Wizard](mpp-create-lib.html). If you need to add a target 
+later, you can do this manually using target presets for [supported platforms](mpp-supported-platforms.html).
+
+Learn more about [additional settings for targets](mpp-dsl-reference.html#common-target-configuration).
```

Comment:
As compilations are a crucial part of targets, it doesn't look right to me that this page entirely omits compilations, and so does the DSL reference that this link points to.

Reply:
Mentioned compilations and added a link to read more.